spec/support/page_objects/trainees/show_page.rb

```diff
@@ -0,0 +1,7 @@
+module PageObjects
+  module Trainees
+    class ShowPage < PageObjects::BasePage
```

Comment:
I created this as the summary page https://github.com/DFE-Digital/register-trainee-teacher-data/blob/e1e292500a2681665d9310e6a1133d8f73165366/spec/support/page_objects/trainees/summary_page.rb#L5. Which should we go with? coin flip? 😄

Reply:
I'm easy on this at this stage. I think its function (and possibly url structure) are going to change quite a bit based on the new designs we saw this morning so 🤷

Reply:
I'm basing the page_object on the app/view structure rather than by function or url. I find it easier to find the page object if I know which view I'm working on and visa-versa.